2021-02-05 16:39:37,554 INFO    MainThread:8855 [wandb_setup.py:_flush():70] setting env: {}
2021-02-05 16:39:37,554 INFO    MainThread:8855 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-02-05 16:39:37,554 INFO    MainThread:8855 [wandb_setup.py:_flush():70] setting login settings: {'api_key': '6d802b44b97d25931bacec09c5f1095e6c28fe36'}
2021-02-05 16:39:37,554 INFO    MainThread:8855 [wandb_setup.py:_flush():70] setting login settings: {'api_key': '6d802b44b97d25931bacec09c5f1095e6c28fe36'}
2021-02-05 16:39:37,554 INFO    MainThread:8855 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-05 16:39:37,555 INFO    MainThread:8855 [wandb_init.py:_log_setup():318] Logging user logs to /home/wolf/MyCode/HPC/wandb/run-20210205_163937-1u9ccxyn/logs/debug.log
2021-02-05 16:39:37,555 INFO    MainThread:8855 [wandb_init.py:_log_setup():319] Logging internal logs to /home/wolf/MyCode/HPC/wandb/run-20210205_163937-1u9ccxyn/logs/debug-internal.log
2021-02-05 16:39:40,567 INFO    MainThread:8855 [wandb_run.py:_console_start():1399] atexit reg
2021-02-05 16:39:40,568 INFO    MainThread:8855 [wandb_run.py:_redirect():1262] redirect: SettingsConsole.REDIRECT
2021-02-05 16:39:40,574 INFO    MainThread:8855 [wandb_run.py:_redirect():1267] Redirecting console.
2021-02-05 16:39:40,574 INFO    MainThread:8855 [redirect.py:install():213] install start
2021-02-05 16:39:40,576 INFO    MainThread:8855 [redirect.py:install():228] install stop
2021-02-05 16:39:40,576 INFO    MainThread:8855 [redirect.py:install():213] install start
2021-02-05 16:39:40,578 INFO    MainThread:8855 [redirect.py:install():228] install stop
2021-02-05 16:39:40,578 INFO    MainThread:8855 [wandb_run.py:_redirect():1313] Redirects installed.
2021-02-05 16:39:47,537 INFO    MainThread:8855 [wandb_watch.py:watch():39] Watching
2021-02-05 16:39:47,538 ERROR   MainThread:8855 [util.py:get_module():167] Error importing optional module torch
Traceback (most recent call last):
  File "/home/wolf/miniconda3/envs/wolf/lib/python3.8/site-packages/wandb/util.py", line 162, in get_module
    return import_module(name)
  File "/home/wolf/miniconda3/envs/wolf/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'torch'
2021-02-05 16:39:47,545 INFO    MainThread:8855 [wandb_run.py:_atexit_cleanup():1369] got exitcode: 1
2021-02-05 16:39:47,546 INFO    MainThread:8855 [wandb_run.py:_restore():1341] restore
2021-02-05 16:39:47,546 INFO    MainThread:8855 [redirect.py:uninstall():232] uninstall start
2021-02-05 16:39:47,546 INFO    MainThread:8855 [redirect.py:_stop():287] _stop: stdout
2021-02-05 16:39:47,546 INFO    MainThread:8855 [redirect.py:_stop():293] _stop closed: stdout
2021-02-05 16:39:47,546 INFO    stdout    :8855 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-02-05 16:39:47,546 INFO    stdout    :8855 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-02-05 16:39:47,546 INFO    MainThread:8855 [redirect.py:_stop():299] _stop joined: stdout
2021-02-05 16:39:47,547 INFO    MainThread:8855 [redirect.py:_stop():301] _stop rd closed: stdout
2021-02-05 16:39:47,547 INFO    MainThread:8855 [redirect.py:uninstall():236] uninstall done
2021-02-05 16:39:47,547 INFO    MainThread:8855 [redirect.py:uninstall():232] uninstall start
2021-02-05 16:39:47,547 INFO    MainThread:8855 [redirect.py:_stop():287] _stop: stderr
2021-02-05 16:39:47,547 INFO    stderr    :8855 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-02-05 16:39:47,547 INFO    stderr    :8855 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-02-05 16:39:47,547 INFO    MainThread:8855 [redirect.py:_stop():293] _stop closed: stderr
2021-02-05 16:39:47,547 INFO    MainThread:8855 [redirect.py:_stop():299] _stop joined: stderr
2021-02-05 16:39:47,547 INFO    MainThread:8855 [redirect.py:_stop():301] _stop rd closed: stderr
2021-02-05 16:39:47,547 INFO    MainThread:8855 [redirect.py:uninstall():236] uninstall done
2021-02-05 16:39:47,780 INFO    MainThread:8855 [wandb_run.py:_wait_for_finish():1492] got exit ret: file_counts {
  wandb_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 5391
  total_bytes: 5391
}

2021-02-05 16:39:49,783 INFO    MainThread:8855 [wandb_run.py:_wait_for_finish():1492] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 9012
  total_bytes: 9012
}

2021-02-05 16:39:51,192 INFO    MainThread:8855 [wandb_run.py:_show_files():1714] logging synced files
2021-02-05 16:39:51,198 INFO    MainThread:8855 [internal.py:handle_exit():156] Internal process exited
